core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -807,6 +807,12 @@ public MetricKey build() {
           .setMetricType(MetricType.COUNTER)
           .setIsClusterAggregated(false)
           .build();
+  public static final MetricKey WORKER_BLOCKS_EVICTED_RATE =
+      new Builder(Name.WORKER_BLOCKS_EVICTED_RATE)
+          .setDescription("Block eviction rate in this worker.")
+          .setMetricType(MetricType.METER)
+          .setIsClusterAggregated(false)
```

Comment:
@yuzhu Should this metric (Eviction rate) be per-worker or across the entire cluster?

Reply:
if you set ClusterAggregated to true, it will produce an aggregated cluster metric at the master.

Reply:
as well as per worker stats

Reply:
So is there any reason to not have it be `true` in this case? Is there a performance overhead or is that negligable?

Reply:
yeah it will increase the metrics aggregation rpc that is sent to the master.  but i think in this case, we want the aggregate eviction rate, so we would want it set to true.